pages/movies.js

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Button,
+  CardActionArea,
+  CardActions,
+  Grid,
+} from "@mui/material";
+import dbConnect from "../lib/dbConnect";
+import Movie from "../models/movie";
+
+/* TODO:
+- Refine styling of MUI cards and grid -- symmetrical etc
+  - Set max height for Title text area
+
+  Optional:
+  - Make cards responsive i.e. 4 per row on mobile
+  - Change font style to correspond with rest of site
+  - Change CardContent bg color to match site
+  - hover effect on cards to stand out
+  - Animate on hover
+  - Link Book to bookings page
+---
+DONE:
+- Display description and title from db: - DONE
+  - movie.description - DONE - DELETED
+  - movie.title - DONE
+- Import MUI grid - DONE
+- Link cards to correct movie page:
+  - href={`/movies/${encodeURIComponent(movie.id)} per https://nextjs.org/docs/api-reference/next/link - DONE
+- Refine styling
+  - Center cards and 4 per row -- DONE
+  - Make Title text a bit smaller -- DONE
+
+- Questions...
+  - Do changes to _app.js and addition of _document.js pose any risk for merge conflicts?
+    - unsure if links should be used in some special way when combining Next + MUI per https://mui.com/material-ui/guides/routing/#more-examples
+*/
+
+export default function MoviesPage({ movies }) {
+  return (
+    <>
+      <Grid
+        container
+        spacing={2}
+        sx={{ maxWidth: "1000px", mx: "auto" }}
+      >
+        {movies.map((movie) => (
+          <div key={movie._id}>
```

Comment:
Nice useage of the mongoDB document _id to keep all keys unique!

Reply:
Thanks, Instead of re-inventing the wheel, I found a good "wheel design" made by a pretty good engineer actually! ;D